test/e2e/manifests/pkg/dependency-upgrade/version/configuration-initial.yaml

```diff
@@ -3,5 +3,5 @@ kind: Configuration
 metadata:
   name: configuration-depends-on-version
 spec:
-  # TODO: Push these to xpkg.upbound.io/crossplane/e2e-... and include package/crossplane.yaml
-  package: xpkg.upbound.io/ezgiorg/ezgi-configuration-nop:v0.1.0
\ No newline at end of file
+  # NOTE(ezgidemirel): This package is manually built and pushed to the registry.
```

Comment:
Could you also put package.yaml of this for future reference.

Reply:
done